spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTM2.scala

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn
+
+import com.intel.analytics.bigdl.nn.Graph.ModuleNode
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity, TensorModule}
+import com.intel.analytics.bigdl.optim.Regularizer
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+
+import scala.reflect.ClassTag
+
+/**
+ * Long Short Term Memory architecture similar to nn/LSTM but without Dropout
+ * It is used to verify the inference results of mkldnn/LSTM
+ */
+// @SerialVersionUID(- 8176191554025511686L)
+class LSTM2[T : ClassTag] (
```

Comment:
why LSTM2? we need a better name

Reply:
LSTMSpec has been modified and LSTM2 is not used anymore. Old LSTM is enough.